Makefile

```diff
@@ -64,21 +66,23 @@ docs: clean-docs
 release: clean assets
 	python setup.py sdist upload
 	python setup.py bdist_wheel upload
+	@echo ""
+	@echo "Now run something like twine -s dist/* to upload all results in the dist/ folder"
```

Comment:
I manually specify which GPG key to use for signing, that's why I don't want it... but also, I think it's better to read the output in the console before uploading.. once a version is released on PyPi, you can't go back.. there's no way of re-doing a release, you have to bump the version.